shared/middleware/helpers/storage_id.rb

```diff
@@ -63,7 +63,12 @@ def storage_encrypt_app_id(storage_id, app_id)
 
 def storage_id(endpoint)
```

Comment:
when you switch to separate NoSQL tables for each of these, you'll have to move the arguments around. so this change is less churn to get us to the 27th.